free-courses-ru.md

```diff
@@ -1,8 +1,8 @@
-### Index
+### Cодержание
 
 * [C++](#C++)
 * [Clojure](#clojure)
-* [Design and Architecture](#design-and-architecture)
+* [Design and Architecture-Дизайн и Aрхитектура](#design-and-architecture)
```

Comment:
Not 100% sure, but I think the table of contents should look like this!
Ugly as hell, but the English title won't work anymore if it's not in English, so we need to URL encode the Russian.

```suggestion
* [Дизайн и Aрхитектура](#%D0%94%D0%B8%D0%B7%D0%B0%D0%B9%D0%BD%20%D0%B8%20A%D1%80%D1%85%D0%B8%D1%82%D0%B5%D0%BA%D1%82%D1%83%D1%80%D0%B0)
```

Reply:
Also, the link probably breaks.